phiml-blog-supporting-code/FNO/fourierLayer.m

```diff
@@ -0,0 +1,27 @@
+function layer = fourierLayer(tWidth,numModes,args)
+
+arguments
+    tWidth
+    numModes
+    args.Name = ""
+end
+name = args.Name;
+
+net = dlnetwork;
+
+layers = [
+    identityLayer(Name="in")
+    spectralConvolution1dLayer(tWidth,numModes,Name="specConv")
```

Comment:
I would prefer the arguments here to be the other way around, `fourierLayer(numModes, tWidth)` and `spectralConvolution1dLayer(numModes, tWidth)`. I think that has a better analogy to `convolution1dLayer(filterSize, numFilters)`.

The `numFilters` and `tWidth` are analogous in that they control some hidden size of some linear operation with the weights. The `numModes` and `filterSize` are analogous in that they control the size/resolution of the convolution filter in some sense, and both can have N entries for an N-d conv or spectral conv.